src/main/java/net/minecraftforge/fluids/Fluid.java

```diff
@@ -110,6 +110,7 @@
      *
      * Generally this is associated with negative density fluids.
      */
+    @Deprecated
```

Comment:
I think it should be replaced with `isLighterThanAir()` which does the density check.